platform/config-core/src/main/java/org/motechproject/config/core/utils/ConfigPropertiesUtils.java

```diff
@@ -115,6 +117,31 @@ public static Properties getPropertiesFromSystemVarString(String string) {
     }
 
     /**
+     * Returns {@code PropertiesConfiguration} and creates config file if does not exist
+     * @param basePath path to the file with config
+     * @param fileName name of the file with config
+     */
+    public static PropertiesConfiguration getPropertiesConfiguration(String basePath, String fileName) {
```

Comment:
Main purpose of this method is to return PropertiesConfiguration. Create a file is only an utility method for this. I moved this creation to another method for readability.